version.py

```diff
@@ -34,7 +34,7 @@
 # new level means in api_docs/changelog.md, as well as "**Changes**"
 # entries in the endpoint's documentation in `zulip.yaml`.
 
-API_FEATURE_LEVEL = 355  # Last bumped for adding 'ignored_because_not_subscribed_channels'
+API_FEATURE_LEVEL = 356  # Last bumped for adding 'ignored_because_not_subscribed_channels'
```

Comment:
Usually, we change the string here. But I guess with the new `unmerged.d` feature for managing API changelog work, this won't be touched by humans anyway soon.

Reply:
Sorry, just forgot to update it. Pushing a fix now.